docs/en/ingest-management/elastic-agent/configuration/providers/kubernetes-provider.asciidoc

```diff
@@ -68,12 +68,17 @@ will be excluded from the event.
 By default it is `true`.
 `annotations.dedot`:: (Optional) If set to be `true` in the provider config, then `.` in annotations will be replaced
 with `_`. By default it is `true`.
-`add_resource_metadata`:: (Optional) Specify labels and annotations filters for the extra metadata coming from node and namespace.
- `add_resource_metadata` can be done for `node` or `namespace`. By default all labels are included
-while annotations are not. These settings are useful when storing labels and annotations
-that require special handling to avoid overloading the storage output. The enrichment of `node` or `namespace` metadata
-can be individually disabled by setting `enabled: false`.
-Example:
+
+`add_resource_metadata`:: (Optional) Specify filters and configration for the extra metadata, that will be added to the event.
+Configuration parameters:
+ * `node` or `namespace`: Specify labels and annotations filters for the extra metadata coming from node and namespace. By default
+ all labels are included while annotations are not. To change default behaviour `include_labels`, `exclude_labels` and `include_annotations`
+ can be defined. Those settings are useful when storing labels and annotations that require special handling to avoid overloading the storage output.
```

Comment:
```suggestion
 all labels are included while annotations are not. To change the default behaviour `include_labels`, `exclude_labels` and `include_annotations`
 can be defined. These settings are useful when storing labels and annotations that require special handling to avoid overloading the storage output.
```